pyproject.toml

```diff
@@ -46,6 +46,8 @@ typeguard = ">=2.13.3"
 xdoctest = {extras = ["colors"], version = ">=0.15.10"}
 myst-parser = {version = ">=0.16.1"}
 dparse = ">=0.5.2"
+pytest-cov = "^4.0.0"
+pytest-mock = "^3.10.0"
```

Comment:
If all tests were always run in the nox environment, these may not be necessary to include as dev dependencies; however if run with `poetry run pytest` then these will be necessary.